src/components/VAudioTrack/layouts/VBoxLayout.vue

```diff
@@ -53,11 +66,15 @@ export default defineComponent({
 
       return props.size ? `${magnitude}rem` : null
     })
+    const categoryLabel = computed(() =>
```

Comment:
The use of `useContext.i18n` and $t requires different kinds of mocking in jest tests. I used `useContext` to simplify mocking.

Reply:
FWIW you can use the `useI18n` function instead if you'd like from `~/composables/use-i18n` (this would help slightly with the Nuxt bridge upgrade as well)

Reply:
Thank you for your advice on what's best for Nuxt bridge upgrade! I'll try changing now.
Just a note: some test files appear that they set up i18n, but actually it doesn't fully work, and renders the `key`s of the $('t') expressions.

Reply:
It may be easier for us to globally mock it in the jest setup file somehow, if that's possible :thinking: Maybe we need a factory function for the vue instance to use or something? :shrug: